htsinfer/get_read_layout.py

```diff
@@ -20,7 +20,8 @@
 
 
 class GetReadLayout:
-    """Determine the adapter sequence present in the FASTQ sequencing libraries.
+    """Determine the adapter sequence present in the FASTQ sequencing
+    libraries.
```

Comment:
This also doesn't have anything to do with the test strategy.

Reply:
I created another PR for this if that's okay.